### DISCUSSION TITLE
Guid Data Type

### DISCUSSION THREAD

**Discussion by kmxsoft** [pawelsalawa/sqlitestudio#3905] [Ideas]
I just found this software, and would like to congratulate the author on a job well done.

One small enhancement request - would be nice if the available tble coumn types included GUID.

**Comment by houghtonap** [comment]
Unfortunately you would need to modify the internals of SQLite to have a new column type for GUID/UUID and I don't believe that will happen. However, you can create a blob column and assign the value of the uuid_blob() extension function and accomplish the same thing.
